app/controllers/registrations_controller.rb

```diff
@@ -26,6 +26,15 @@ def disable_two_factor
     end
   end
 
+  def tour
```

Comment:
You need to add the actions to this policy:
https://github.com/Codeminer42/cm42-central/blob/master/app/policies/user_policy.rb#L20

I suggest using the check `is_himself?`. Also, as this action updates only the current user, you should use `current_user` instead of `@user = User.find(params[:id])`